core/src/main/scala/cromwell/core/ExecutionStatus.scala

```diff
@@ -2,13 +2,15 @@ package cromwell.core
 
 object ExecutionStatus extends Enumeration {
   type ExecutionStatus = Value
-  val NotStarted, QueuedInCromwell, Starting, Running, Failed, Preempted, Done, Aborted = Value
-  val TerminalStatuses = Set(Failed, Done, Aborted, Preempted)
+  val NotStarted, QueuedInCromwell, Starting, Running, Failed, Preempted, Done, Esled, Aborted = Value
+  val TerminalStatuses = Set(Failed, Done, Aborted, Preempted, Esled)
 
   implicit class EnhancedExecutionStatus(val status: ExecutionStatus) extends AnyVal {
     def isTerminal: Boolean = {
       TerminalStatuses contains status
     }
+
+    def isDone: Boolean = status == Done || status == Esled
```

Comment:
Hmmm, since this method encompasses jobs that are done or esled, can it it have a name other than "isDone". When I saw it being used by the WorkflowExecutionActor I assumed it was simply execution statuses that were done, though that's not the case. I dont have a good suggestion, but my lame name for it would be "isDoneOrElsed"